Code/Tools/ProjectManager/Source/GemCatalog/GemCatalogScreen.cpp

```diff
@@ -327,8 +419,10 @@ namespace O3DE::ProjectManager
         {
             // Add all available gems to the model.
             const QVector<GemInfo>& allGemInfos = allGemInfosResult.GetValue();
-            for (const GemInfo& gemInfo : allGemInfos)
+            for (GemInfo gemInfo : allGemInfos)
             {
+                // Mark as downloaded because this gem was registered with an existing directory
+                gemInfo.m_downloadStatus = GemInfo::DownloadStatus::Downloaded;
                 m_gemModel->AddGem(gemInfo);
```

Comment:
You can move the `AZStd::move(gemInfo)` into the AddGem call to avoid making another copy.

Reply:
`void GemModel::AddGem(const GemInfo& gemInfo)`
AddGem takes the GemInfo by constant reference so there shouldn't be another copy call

Reply:
That did give me an idea to avoid making a copy call. The GemInfos in the result are generated so I can probably just edit them in place in the QVector. They are never used afterward. Will just call out they are being edited.